components/robotsports/getball-fetch/README.adoc

```diff
@@ -0,0 +1,35 @@
+= GetBallFetch Robotsports Variant
+
+== Summary
+
+GetBallFetch is an action that direct the robot to the ball till it is in the beak of the robot.
+
+== Scope and context
+
+This action implements the sub-action getball-fetch (get a stationary ball) of the getball action.
+This variant is RobotSport variant of getball-fetch.
+
+== Interface details
+
+* **purpose**: get the ball
+* **inputs**: See link:./interface/Input.proto[Input.proto] for the exact definition.
+	** WorldState  
+
+* **outputs**:  See link:./interface/Output.proto[Output.proto] for the exact definition.  
+	** robot target set-point (type posvel in FCS) [to PathPlanning]
+	** actionResult [to Reasoning]
+	** estimated time to get the ball (s), for query/decisioning purpose [to Reasoning]
+
+* *parameters*:  
+The action has some configuration parameters. These parameters are specific for this variant of get ball fetch, 
+see link:./interface/Params.proto[Params.proto] and 
+link:./interface/DefaultParams.json[DefaultParams.json].
+
+== Design
+
+Calculate setpoint to fetch the ball. Using the current ball position and extrapolate it by velocity * parameter "vision_delay".
+
```

Comment:
I seem to have noticed the behavior includes a nice feature to ensure rotation is finished, as to prevent pushing the ball away with robot side/back? 

We at Falcons do not have that (because we have a design rule to always look at ball, so we never bothered, but in basic tests/scripts it would be a nice addition.

So if your design includes it, then it might be interested to mention this in README and demonstrate with a test (which would fail on the Falcons version).

Reply:
Interesting observation.  The falcons design rule makes the component less independent then the Robotsports variant. The Falcons variant requires that user added own rotation update to ensure Rz is finished (look to the ball).  I guess that this missing part effects the performance if it is not handled by the user. Robotsports assumes it is solved by the action. 
I will add the comment and make at least one additional test-case.

Reply:
Added the remark and new test-case.
Did not see the noticed difference in code. The MRA variant of falcons also set a Rz setpoint. This might be different than the non MRA variant.

Reply:
It could also be that the behavior is coming from your velocity controller.

Just checked in Falcons simulator, using RobotSports getball and RobotSports velocityControl, and indeed it first finishes rotation before moving forward to get the ball successfully. I could have put that in the TC demo. 

Anyway I will merge.